README.md

```diff
@@ -6,13 +6,14 @@
 
 ```bash
 git clone https://github.com/opencontainers/runc
```

Comment:
Probably also want to mention cloning into  $GOPATH/opencontainers else you'll get build errors.